### ISSUE TITLE
Any plans to merge with xcodebuild-rb?

### ISSUE THREAD

**Issue by subdigital** [lukeredpath/betabuilder#38]
Seems like the building & parsing output of `xcodebuild-rb` could be useful here, which would leave beta builder simply to package the archives & publish to various places.

I'm guessing this has been discussed before, but I didn't see any issues.

**Comment by subdigital** [comment]
And by merge, I really mean....  make `betabuilder` reference `xcodebuild-rb` to delegate build functionality

**Comment by lukeredpath** [comment]
Yes, it was in the back of my mind. I'm definitely considering it.
